src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.csproj

```diff
@@ -30,8 +37,17 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(OS)'=='Windows_NT'">
-    <None Include="$(OutputPath)net461\win7-x64\$(AssemblyName).exe" Pack="true" PackagePath="build\netstandard2.0\$(AssemblyName)-x64.exe" />
-    <None Include="$(X86ProjectDirectory)\bin\$(Configuration)\net461\win7-x86\$(AssemblyName)-x86.exe" Pack="true" PackagePath="build\netstandard2.0\$(AssemblyName)-x86.exe" />
+    <None Include="$(OutputPath)net461\win7-x86\$(AssemblyName)-x86.exe" Pack="true" PackagePath="build\netstandard2.0\" />
+    <None Include="$(OutputPath)net461\win7-x64\$(AssemblyName)-x64.exe" Pack="true" PackagePath="build\netstandard2.0\" />
   </ItemGroup>
 
+  <Target Name="BuildX64" AfterTargets="Build" Condition="'$(TargetFramework)'=='net461' AND '$(PlatformTarget)'!='x64'">
```

Comment:
Can you help me understand this part? If the PlatformTarget isn't `x64`, build `x64`.

Reply:
It's to prevent the x64 build from kicking off another x64 build.